pkg/pubsubx/worker.go

```diff
@@ -200,15 +201,27 @@ func (s *Subscription) Receive(ctx context.Context, st *query.Stores) error {
 func (s *Subscription) ProcessMessage(ctx context.Context, db *query.Stores, msg *Message) error {
 	switch {
 	case strings.HasSuffix(msg.EventType, ".added"):
-		resource, err := query.NewResourceFromURN(msg.SubjectURN)
+		subjectURN, err := urnx.Parse(msg.SubjectURN)
+		if err != nil {
+			fmt.Println("Error getting resource from URN for subject")
```

Comment:
why are we `fmt`in' when we have a logger?

Reply:
and below

Reply:
Should be fixed now!